### PR TITLE
More SUSE compliance

### PR DESCRIPTION
**Pull request by jirihnidek**
* Original PR provided by Philipp Seiler <p.seiler@linuxmail.org>
  - PR: #2842
  - Commit: 1fd20198fd8c4f8e9e1b84484e06d75858ca4b38
* Simplified this PR due to another PR provided by Pino #2854
* Fixed few issues from origin PR
* SUSE specific parts are grouped together
* We build only 4 packaged for SUSE. Thus it was possible to remove
  {suse_version} from many places, because it was useless

### CODE REVIEW

File: subscription-manager.spec [hunk 1]
```diff
@@ -563,25 +543,30 @@ find %{buildroot} -name \*.py* -exec touch -r %{SOURCE0} '{}' \;
 %files -f rhsm.lang
 %defattr(-,root,root,-)
 
-# Make some unusual directories for suse part of subscription-manager
+# Make some unusual directories amd files for suse part of subscription-manager
```
Comment (sbernhard) [inline subscription-manager.spec:546]: typo: amd -> and

**Comment by sbernhard** [comment]
I tried to build this for SLES 15 with the OpenSuse Build Service. 
First of all, it looks pretty good. Just some small enhancements are necessary to get it build:

```
--- subscription-manager.spec.old	2021-12-21 17:57:33.470275650 +0100
+++ subscription-manager.spec	2021-12-21 17:53:24.212505906 +0100
@@ -135,6 +135,11 @@
 Source1: %{name}-cockpit-%{version}.tar.gz
 %endif
 
+# Especially for the OpenSuse Build Service we need to have another lint config
+%if 0%{?suse_version}
+Source2: subscription-manager-rpmlintrc
+%endif
+
 # The following macro examples are preceeded by '%' to stop macro expansion
 # in the comments. (See https://bugzilla.redhat.com/show_bug.cgi?id=1224660 for
 # why this is necessary)
@@ -218,6 +223,7 @@
 BuildRequires: distribution-release
 BuildRequires: libzypp
 BuildRequires: systemd-rpm-macros
+BuildRequires: python3-rpm-macros
 BuildRequires: %{py_package_prefix}-python-dateutil
 %else
 BuildRequires: system-release
```

See: https://build.opensuse.org/package/show/home:orcharhino/subscription-manager

**Review by sbernhard — CHANGES_REQUESTED** [review]

**Review by sbernhard — APPROVED** [review]
LGTM. Thanks!

**Comment by jirihnidek** [comment]
We agreed that it's OK to merge to main.

**Pull request merged**